source/src/ports/generic_networkhandler.c

```diff
@@ -256,7 +256,7 @@ EipStatus NetworkHandlerInitialize(void) {
    * A newly accept()ed TCP socket inherits the setting from this socket.
    */
   if (SetQosOnSocket( g_network_status.tcp_listener,
-                     GetPriorityForSocket(0xFFFF) ) != 0) {
+                     CipQosGetDscpPriority(0xFFFF) ) != 0) {
```

Comment:
replace magic number

Reply:
as already commented before the code lacks kConnectionObjectPriorityExplicit